plugins/rest/help.md

```diff
@@ -104,21 +104,19 @@ This action is used to make a PUT request.
 
 |Name|Type|Default|Required|Description|Enum|Example|
 |----|----|-------|--------|-----------|----|-------|
-|body|object|None|False|Payload to submit to the server when making the HTTP Request call|None|{"user": "user@example.com"}|
-|body_as_an_array|[]object|None|False|Payload (Array) to submit to the server when making the HTTP Request call|None|[{"user": "user@example.com"}]|
+|body_any|string|None|False|Payload (non-dictionary) to submit to the server when making the HTTP Request call|None|Any type can be here, ints, arrays, you name it!|
```

Comment:
I think there should be an example here to show what could be put in here. You could add the clarification of any type to the description